### PR TITLE
Radiopack and Radioman

### PR DESCRIPTION
**Pull request by bardinatavern4**
Implements the existing radiopack (thanks Popoff) and adds the Radioman job.

### CODE REVIEW

File: mojave/jobs/job_types/ncr/radioman.dm [hunk 1]
```diff
@@ -0,0 +1,29 @@
+/datum/job/ms13/ncr/radioman
+	title = "NCR Radioman"
+	total_positions = 4
+	spawn_positions = 4
+	supervisors = "Sergeant, and Lieutenant"
```
Comment (uomo91) [inline mojave/jobs/job_types/ncr/radioman.dm:5]:
```suggestion
	supervisors = "the Corporals, the Sergeants, and the Lieutenant"
```

File: mojave/jobs/job_types/ncr/radioman.dm [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+	description = "Relay messages, obey orders and relay changes in situations to keep command up to date."
+	forbid = "The NCR forbids: Recreational drugs(barring med-x), drinking or gambling on duty, disobeying lawful orders. CO determines who is off-duty. Radiomen found to be abusing their radio will be first delimbed, then shot."
```
Comment (uomo91) [inline mojave/jobs/job_types/ncr/radioman.dm:7]: `forbid` should be generic and it should be set on the base NCR job, see the Legion for an example.

File: mojave/jobs/job_types/ncr/radioman.dm [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+	enforce = ""
```
Comment (uomo91) [inline mojave/jobs/job_types/ncr/radioman.dm:8]: Move what you wrote in `description` to here, I think.

**Review by uomo91 — CHANGES_REQUESTED** [review]

**Review by uomo91 — CHANGES_REQUESTED** [review]
You need to put `#include "mojave\jobs\job_types\ncr\radioman.dm"` in tgstation.dme

**Comment by bardinatavern4** [comment]
Forgot to include the .dme, but it's there now. Short list of changes from first PR:

slots decreased to two,
fixed capitalizations as per uomo
some of the flavor text changed as per uomo
Can no longer hold more then 5 items + walkie talkie, meaning it only holds one in game lines worth
Should be good to go

**Review by uomo91 — APPROVED** [review]
Approved for now, some planned fixes to things in the future.

**Pull request merged**